cmd/tailscale/cli/debug.go

```diff
@@ -784,6 +790,29 @@ func runDaemonLogs(ctx context.Context, args []string) error {
 	}
 }
 
+func runDaemonBusEvents(ctx context.Context, args []string) error {
+	logs, err := localClient.TailBusEvents(ctx)
+	if err != nil {
+		return err
+	}
+	d := json.NewDecoder(logs)
```

Comment:
Please make sure the body gets closed.
```suggestion
	defer logs.Close()
	d := json.NewDecoder(logs)
```

I wonder if we should also buffer the reader, since the bytes are coming in off a TCP connection
```suggestion
	d := json.NewDecoder(bufio.NewReader(logs))
```

I see that the `json.Decoder` does keep a few bytes around for token parsing, but it otherwise I think reads byte-by-byte.